operator/api/pulumi/shared/stack_types.go

```diff
@@ -159,6 +159,13 @@ type StackSpec struct {
 	// The default behavior is to retain the workspace. Valid values are one of "Retain" or "Delete".
 	// +optional
 	WorkspaceReclaimPolicy WorkspaceReclaimPolicy `json:"workspaceReclaimPolicy,omitempty"`
+
+	// (optional) PulumiLogVerbosity is the log verbosity level to use for the Pulumi CLI.
+	// If unset,verbose logging is disabled.
+	// See: https://www.pulumi.com/docs/iac/support/troubleshooting/#verbose-logging
+	// for more information about log levels.
+	// +optional
+	PulumiLogVerbosity uint32 `json:"pulumiLogLevel,omitempty"`
```

Comment:
I think we resisted adding first-class fields when it simply copies what's possible in template.  Let's just leave it at the template level, WDYT?

Reply:
It does make it easier to manage. Will update.

Reply:
That's fair. We can always add this top level field in the future. Always easier to add, than remove :)